students/luke/s07/html_render.py

```diff
@@ -1,17 +1,90 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 
 """
 A class-based system for rendering html.
 """
 
 # This is the framework for the base class
 class Element(object):
+    tag = ''
+
+
+    def __init__(self, content=None, style=None):
+        self.content = []
```

Comment:
Can you add args and kwargs here to increase functionality, keeping in mind that you can use args as a list, and kwargs like a dictionary?